pkg/windows/service/service.go

```diff
@@ -21,7 +21,9 @@ package service
 
```

Comment:
I would suggest to put an OWNERS file in this directlory so SIG Windows can approve it.

If you do so, please also include SIG Node team as approvers

Reply:
thanks, this is also what we discussed before, let me check with @marosset and @jsturtevant

Reply:
Please make sure to add Node reviewers and approvers into that file since it will be Node-related code. I don't think we should block every change in this file by `pkg/` approvers.

Reply:
Added Owner file @SergeyKanzhelev